handsontable/src/pluginHooks.js

```diff
@@ -571,9 +573,11 @@ const REGISTERED_HOOKS = [
   'afterRenderer',
 
   /**
-   * Fired after changing a row sequence. It populates every change on indexes captured by {@link IndexMapper}.
+   * Fired after the order of rows has changed.
+   * This hook is fired by changing row indexes of any type supported by the {@link IndexMapper}.
    *
-   * @param {'init'|'remove'|'insert'|'move'|'update'} [source] String that identifies source of row sequence change.
```

Comment:
@kirszenbaum 
Is there a reason why you changed the `@param` type from a list of possible values to `{string}`?
I can't find it in the JSDoc documentation, but I _think_ it's a valid syntax 🤔

Reply:
@jansiegel it works, but it renders, for example, like this ([source](https://github.com/handsontable/handsontable/blob/develop/handsontable/src/dataMap/dataMap.js#LL309C6-L309C6) > [output](https://handsontable.com/docs/javascript-data-grid/api/data-map/#createrow)):

<img width="500" alt="2" src="https://github.com/handsontable/handsontable/assets/29247866/13dd5e10-8c3f-4cc1-a9fd-37ba905363be">

I guess we should keep the syntax and create for rendering it properly?

Reply:
@jansiegel @wszymanski ok, done: [3a2f19a](https://github.com/handsontable/handsontable/pull/10374/commits/3a2f19aba66e28fbe3929555bcaa336b370312b6)

Reply:
@AMBudnik – that's the expected outcome (as discussed [here](https://github.com/handsontable/handsontable/pull/10374#discussion_r1200116160)). The JSDocs syntax is correct but we need to fix the way it's rendered in the docs. I'll create an issue for that.